.goreleaser.yaml

```diff
@@ -98,14 +98,22 @@ brews:
 
 dockers:
   - image_templates:
-      - "anchore/syft:latest"
-      - "anchore/syft:{{ .Tag }}-amd64"
-      - "anchore/syft:v{{ .Major }}-amd64"
-      - "anchore/syft:v{{ .Major }}.{{ .Minor }}-amd64"
-      - "ghcr.io/anchore/syft:latest"
-      - "ghcr.io/anchore/syft:{{ .Tag }}-amd64"
-      - "ghcr.io/anchore/syft:v{{ .Major }}-amd64"
-      - "ghcr.io/anchore/syft:v{{ .Major }}.{{ .Minor }}-amd64"
+      - anchore/syft:{{.Tag}}-debug
```

Comment:
should we be providing a `anchore/syft:latest-debug` ? Or I guess it makes more sense as `anchore/syft:debug`

Reply:
I can add `anchore/syft:debug` as the rolling latest version here